### ISSUE TITLE
WARNING:bacpypes.bvllservice.UDPMultiplexer:unsupported message

### ISSUE THREAD

**Issue by ChristianTremblay** [JoelBender/bacpypes#7]
Message are correctly sent to the network (verified with Wireshark)
Response is misinterpreted somewhere and give the warning...

Debug : 
```
WARNING:bacpypes.bvllservice.UDPMultiplexer:unsupported message
    pduSource = <Address 2:b'\x05'>
    pduExpectingReply = 0
    pduNetworkPriority = 0
    apduType = 7
    apduSrv = False
    apduInvokeID = 1
    apduAbortRejectReason = 65
    pduData = x''
```

**Comment by JoelBender** [comment]
This is actually issue #7, I removed the issue name from the title.

**Comment by JoelBender** [comment]
The message byte should have been unpacked as unsigned, fixed in the issue-7 branch so far.  To be continued.

**Comment by JoelBender** [comment]
I added test code in the **tests/test_pdu/test_address.py** file that pushes the edges of `LocalStation`, `LocalBroadcast`, and `GlobalBroadcast` a little to get the in swing of writing tests.  I suspect that I will have to abandon py25 testing and stick with py27 and py34, the `with self.assertRaises()` isn't available because the `with` statement is in py26.

Note that I added it to the *issue-5* branch by mistake, groan.  *Issue-5* has been merged into *issue-7* and is waiting for some testing before being merged into *master*.
